index.html

```diff
@@ -0,0 +1,80 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/06aa7828ae.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+<!-- <header class="toolbar">
+  <a href="#" class="toolbar logo"><img src="images/logo.png" alt=""></a>
+  <nav class="toolbar menu"><img src="images/menu.png" alt=""></nav>
+</header>
+<section class ="container">
+  <div class="bg-image1"></div>
+  <div class="bg-image2"></div>
+  <div class="bg-image3"></div>
+  <div class="bg-image4"></div>
+  <div class="bg-image6"></div>
+  <div class="bg-image7"></div>
+  <div class="bg-imagea"></div>
+  <div class="bg-imageb"></div> -->
+     <!-- <div class="intro"><h1>I'm Victor,
+       Glad to see you!</h1></div>
+          <div class="content">
+           <p>Hello! I am a software developer! I can help you build a product, feature or website.
+           Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
+          </div>
+          
+     <ul class="social">
+       <li><img src="images/connect.png" alt=""></li>
+     </ul>
+     <ul>
+       <li class="linked"><img src="images/linkedin.png" alt=""></li>
+       <li class="twit"><img src="images/twitter.svg" alt=""></li>
+       <li class="vectorm"><img src="images/vectorm.svg" alt=""></li>
+       <li class="vectorp"><img src="images/vectorp.svg" alt=""></li>
+       <li class="vector"><img src="images/vector.svg" alt=""></li>
+     </ul>
+     
+</section> -->
+
+<header class="header">
+  <a href="#" class="toolbar logo"><img src="images/logo.png" alt=""></a>
+  <nav class="toolbar menu"><img src="images/menu.png" alt=""></nav>
+</header>
+<section class="headline">
+  <div class="bg-icons">
+    <div class="bg">
+      
+    </div>
+    <div class="top">
+      <div class="description">
+        <h1>I'm Victor,
+          Glad to see you!</h1>
+          <p class="my-description">
+            Hello! I am a software developer! I can help you build a product, feature or website.
+                 Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me
+          </p>
+      </div>
+      <div class="connect">
+        <h3>LETS CONNECT</h3>
+        <ul class="social-icons">
+          <li class="icon"><a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+          <!-- <li class="linked"><img src="images/linkedin.png" alt=""></li> -->
+          <li class="icon"><a href=""><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+          <li class="icon"><a href=""><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+          <li class="icon"><a href=""><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+          <li class="icon"><a href=""><i class="fa fa-github" aria-hidden="true"></i></a></li>
+        </ul>
```

Comment:
- [ ] As a previous reviewer mentioned, please export icons from Figma and use them to match the design, here is the [video](https://www.loom.com/share/245c0a81c8054639902abbd623f57054) for how to export icons from template